locales/ara.json

```diff
@@ -1,21 +1,23 @@
 {
   "ActivityLogText": {
-    "VC_SHARED": "مشترك",
-    "VC_RECEIVED": "الواردة",
-    "VC_RECEIVED_NOT_SAVED": "تلقى لم يتم حفظها",
-    "VC_DELETED": "تم الحذف",
-    "VC_DOWNLOADED": "تم تنزيله",
-    "VC_REVOKED": "إلغاء - فسخ",
-    "VC_SHARED_WITH_VERIFICATION_CONSENT": "مشترك. يتم إعطاء الموافقة للتحقق من التواجد",
-    "VC_RECEIVED_WITH_PRESENCE_VERIFIED": "الواردة. تم التحقق من الحضور",
-    "VC_RECEIVED_BUT_PRESENCE_VERIFICATION_FAILED": "الواردة. فشل التحقق من التواجد",
-    "PRESENCE_VERIFIED_AND_VC_SHARED": "التحقق منها ومشاركتها",
-    "PRESENCE_VERIFICATION_FAILED": "فشل التحقق",
-    "QRLOGIN_SUCCESFULL": "QR تم تسجيل الدخول بنجاح",
-    "WALLET_BINDING_SUCCESSFULL": "تفعيل ناجحة",
-    "WALLET_BINDING_FAILURE": "فشل التفعيل",
-    "VC_REMOVED": "تمت إزالته من المحفظة",
-    "TAMPERED_VC_REMOVED": "تمت إزالة بعض البطاقات بسبب نشاط ضار"
+    "VC_SHARED": "تم مشاركة {{idType}} {{id}} بنجاح.",
+    "VC_RECEIVED": "تم استلام {{idType}} {{id}}.",
+    "VC_RECEIVED_NOT_SAVED": "لم يتمكن من حفظ {{idType}} {{id}}.",
+    "VC_DELETED": "تم حذف {{idType}} {{id}} بنجاح.",
+    "VC_DOWNLOADED": "تم تنزيل {{idType}} {{id}}.",
+    "VC_REVOKED": "تم إلغاء {{idType}} {{id}} من قِبل المصدر.",
+    "VC_SHARED_WITH_VERIFICATION_CONSENT": "تم مشاركة {{idType}} {{id}} بالموافقة للتحقق من الحضور.",
+    "VC_RECEIVED_WITH_PRESENCE_VERIFIED": "تم استلام {{idType}} {{id}} ونجح التحقق من الحضور.",
+    "VC_RECEIVED_BUT_PRESENCE_VERIFICATION_FAILED": "تم استلام {{idType}} {{id}} ولكن فشل التحقق من الحضور.",
+    "PRESENCE_VERIFIED_AND_VC_SHARED": "نجح التحقق من الوجه وتم مشاركة {{idType}} {{id}}.",
+    "PRESENCE_VERIFICATION_FAILED": "فشل التحقق من الوجه أثناء محاولة مشاركة {{idType}} {{id}}.",
+    "QRLOGIN_SUCCESFULL": "نجح تسجيل الدخول باستخدام QR باستخدام {{idType}} {{id}}.",
+    "WALLET_BINDING_SUCCESSFULL": "تم تفعيل {{idType}} {{id}} بنجاح.",
+    "WALLET_BINDING_FAILURE": "فشل تفعيل {{idType}} {{id}}.",
+    "VC_REMOVED": "تم إزالة {{idType}} {{id}} من المحفظة.",
+    "TAMPERED_VC_REMOVED": "تم إزالة {{idType}} {{id}} من المحفظة بسبب العبث.",
+    "NATIONAL_ID": "الهوية الوطنية",
```

Comment:
If not needed, can we remove the NATIONAL_ID and INSURANCE_ID

Reply:
This is needed for the story to show the type of VC, this is temporary as later there'd be a better way to tell apart ID-Type when more IDs get supported by Inji.

Reply:
E.g. 

`National ID 1234 downloaded` --> downloaded from E-Signet using OIDC flow / OTP flow
`Insurance Card 1234 downloaded` --> downloaded from OIDC impl at Sunbird side